apps/desktop-ui/src/index.css

```
@@ -616,3 +1154,208 @@ details.openfdd-advanced > summary {
+}
+
+.system-resources-dl dt {
+  margin: 0;
+  font-weight: 600;
+  color: var(--muted);
+  font-size: 12px;
+}
+
+.system-resources-dl dd {
+  margin: 0;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)